addons/website/views/snippets/s_popup.xml

```diff
@@ -59,7 +52,6 @@
                             data-dependencies="s_popup_size_full"/>
         </div>
         <div data-js="SnippetPopup" data-selector=".s_popup" data-exclude="#website_cookies_bar" data-target=".s_popup_main">
-            <we-colorpicker string="Text Color" data-select-style="true" data-css-property="color" data-color-prefix="text-"/>
```

Comment:
I think JKE put that to be able to control the color of the close icon. Maybe we can keep it, rename it to "Close Button Color" and add a data-apply-to="<selector for close button>" ?